### PR TITLE
New Cytology Mobs

### PR DESCRIPTION
**Pull request by Comxy**
## About The Pull Request

This poll request adds:
- 2 new mobs
- 5 new cytology spawns
- 2 new spells
- 1 new innate ability
- 3 now swab-able items

The mobs added to cytology:
- horror mime
- skeleton spirit
- goat
- ghost
- skeleton

## Why It's Good For The Game

It is always fun and good to have more cytology mobs to choose from.

promo_4
promo_5
standing_mime

## Changelog

:cl:
add: Added new mobs to cytology and more!
/:cl:

### CODE REVIEW

**Review by tralezab — CHANGES_REQUESTED** [review]
If oranges doesn't close this, this is what you gotta do to bring it to codebase standards

**Comment by Comxy** [comment]
> 
> 
> if the mobs get a lot more in depth than "stepping on fugu's toes" and "wallphasing andy" i'd be more in support

Can you please give me an example for what you mean with depth? Also I think fugu is a good concept as a creature. It just never gets used as a mob, and only gets used for the buff it gives other creatures. But I see that this might step on the fugu's toes.

**Comment by moocowswag** [comment]
The goliath broodmother also has a self buff ability although it mostly just turns itself red and speeds itself up.

The fugu I believe has inherently super gimped base stats and an exaggerated self buff ability to sell the point home of what its gland does.

And its only used for its gland drop.



The horror mime appears to have some sort of charge that goes on cooldown and apparently its steroid lets it move and attack faster and hit harder? Meanwhile the fugu monster self buff mostly just makes itself tankier and hit harder + move slower

So if we are to compare the mobs themselves and what gameplay as them would be like?

The horror mime is an ultra aggressive glass cannon that cant ever take many hits even while steroided and the steroid just makes it kill people faster and possibly escape for a hit and run but it generally is like an assassin

The fugu is a fast and skittering little critter that wants to position itself as close to its target as possible and quickly buff itself before getting hit in small form and then spends its large phase dealing as much damage as possible and trying to make the situation as safe for itself as it can for when it eventually turns back into critter mode since it lacks the ability to escape combat when enlarged.

File: code/modules/research/xenobiology/vatgrowing/samples/cell_lines/common.dm [hunk 1]
```diff
@@ -649,4 +649,107 @@
 
 	resulting_atoms = list(/mob/living/simple_animal/hostile/jungle/mega_arachnid = 1)
 
+/datum/micro_organism/cell_line/goat
+	desc = "Caprinae stem cells"
+	required_reagents = list(
+	/datum/reagent/consumable/nutriment/protein,
+	/datum/reagent/consumable/nutriment,
+	/datum/reagent/consumable/berryjuice)
```
[the remark this answers is not in the document]
Comment (Comxy) [reply code/modules/research/xenobiology/vatgrowing/samples/cell_lines/common.dm:657]: Cytology lists don't have list entries on the first line.

[the remark this answers is not in the document]
Comment (Comxy) [reply code/modules/research/xenobiology/vatgrowing/samples/cell_lines/common.dm:657]: It is formatted like that for the rest of cytology also.

**Comment by Comxy** [comment]
I tried doing balloons but it didn't work, also I edited some stuff @tralezab.

**Comment by Comxy** [comment]
I know this is not going to go through so I might as well close it now. I will probably start working on something else again in the near future.

**Pull request closed without merging**